src/packet-defs.js

```diff
@@ -721,12 +721,47 @@ export function getPacketDefs(options) {
 				})                       
 			})
 		},
-		
-		
-		//// FIXME: Add subtype 0x0e, presumably generic mesh.
-		//// Sample packet type+subtype+payload:
-		//// fe c8 54 f7 0e 00 00 00 00 00 00 00 94 bb 10 3e
 
+		genericMesh: {
+		  	type: 0x80803df9,
+		  	subtypeLength: 1,
+		  	subtype: 0x0e,	// 14
+		  	fields: type.struct({
+			    	id: type.int32,
+			    	data: type.bitmapstruct(1,{
+					posX:        type.float,
+					posY:        type.float,
+					posZ:        type.float,
+					unknown04:   type.int32,
+					unknown05:   type.int32,
+					unknown06:   type.int64, //Long
```

Comment:
Given that node.js cannot work with 64-bit long integers, I suggest something like the following:

```javascript
unknown06: type.struct({ high: type.int32, low: type.int32 }),
```